storage/innobase/include/fil0crypt.h

```diff
@@ -612,6 +612,59 @@ fil_space_get_scrub_status(
 	const fil_space_t*		space,
 	fil_space_scrub_status_t*	status);
 
+struct redo_log_key {
+	uint  version;
+	char  key[ENCRYPTION_KEY_LEN];
+	ulint read_count;
+	ulint write_count;
+	bool  present;
+
+	bool
+	persisted() const {
```

Comment:
can be one line?

Reply:
It's on two line every other place in the file.